src/Ledger/Conway/Foreign/HSLedger/Utxo.agda

```diff
@@ -29,8 +38,10 @@ utxo-step = to (compute Computational-UTXO)
 
 {-# COMPILE GHC utxo-step as utxoStep #-}
 
-utxow-step : HsType (UTxOEnv → UTxOState → Tx → ComputationResult String UTxOState)
-utxow-step = to (compute Computational-UTXOW)
+utxow-step : ExternalFunctions → HsType (UTxOEnv → UTxOState → Tx → ComputationResult String UTxOState)
+utxow-step ext = to (coerce ⦃ TrustMe ⦄ $ compute Computational-UTXOW)
```

Comment:
I'd go for something like this:
```
module Ext (ext : ExternalFunctions) where
  ts = HSTransactionStructure ext
  af = HSAbstractFunctions ext

module _ (ext : ExternalFunctions) where
  open Ext ext

  open import Ledger.Conway.Conformance.Utxo.Properties ts af using (Computational-UTXO)
  open import Ledger.Conway.Conformance.Utxow.Properties ts af using (Computational-UTXOW)

  utxo-step : HsType (UTxOEnv → UTxOState → Tx → ComputationResult String UTxOState)
  utxo-step = to (coerce ⦃ TrustMe ⦄ $ compute Computational-UTXO)

  utxow-step : HsType (UTxOEnv → UTxOState → Tx → ComputationResult String UTxOState)
  utxow-step = to (coerce ⦃ TrustMe ⦄ $ compute Computational-UTXOW)

{-# COMPILE GHC utxo-step as utxoStep #-}
{-# COMPILE GHC utxow-step as utxowStep #-}
```
As you can see, it scales to multiple definitions quite nicely (and it handles additional structures well if we ever need it). The `Ext` module can also be defined in `Ledger.Conway.Foreign.HSLedger.Core`. For some reason I wasn't allowed to make the two structures instances, maybe there's a bug in instance resolution?